src/js/Utils.js

```diff
@@ -1,40 +1,24 @@
 /**
- * Adapted from https://www.w3schools.com/js/js_cookies.asp
+ * Converts a given value to JSON and writes it to the given key in
+ * localStorage.
  */
-export const setCookie = (key, value, days = 360) => {
+export const setLocalStorage = (key, value) => {
+    localStorage.setItem(key, JSON.stringify(value))
     value = JSON.stringify(value);
```

Comment:
This line can be removed as well, right?

Reply:
That was an oversight on my part. I removed the line in 31443c6 and also added semicolons to adhere to the existing code style better.